building/glass/aptbranch.py

```diff
@@ -73,3 +42,35 @@ def export_key(keyid, armor=False):
     if not result.strip():
         raise Exception("empty result from gpg for keyid: '%s'" % keyid)
     return result
+
+
+def select_branch_config(branches_config: list, branch: str):
+    for config in branches_config:
+        if config["name"] == branch:
+            return config
+    raise Exception("no config found for %s in %s" % (branch, CONFIG_PATH))
+
+
+class Config:
+    def __init__(self, branch: str):
+        if not os.path.exists(CONFIG_PATH):
+            raise Exception("cannot find branches config at %s, use %s.example to create one" % (CONFIG_PATH, CONFIG_PATH))
+        branches_config = validate.load_validated(CONFIG_PATH, CONFIG_SCHEMA_NAME)["branches"]
+        self.config = select_branch_config(branches_config, branch)
+
+        check_signing_key(self.signing_key)
+
+    @property
+    def name(self) -> str:
+        return self.config["name"]
+
+    @property
+    def signing_key(self) -> str:
+        return self.config["signing-key"]
+
+    @property
+    def apt_url(self):
+        url_prefix = self.config.get("apt-url-prefix", "")
+        if url_prefix != "" and not url_prefix.endswith("/"):
+            url_prefix += "/"
```

Comment:
You can just do `url_prefix and ...` instead of `url_prefix != "" and ...` (non-empty strings evaluate to true)